packages/ott-vis/src/components/SimplePanel.tsx

```diff
@@ -48,13 +49,13 @@ export const SimplePanel: React.FC<Props> = ({ options, data, width, height }) =
 				viewBox={`-${width / 2} -${height / 2} ${width} ${height}`}
 			>
 				<g>
-					<circle style={{ fill: theme.colors.primary.main }} r={100} />
+					<ForceGraph />
 				</g>
```

Comment:
`ForceGraph` renders an svg, so putting it here makes it an svg inside another svg. Is this intentional?

Reply:
Not intentional